backend/core/models.py

```diff
@@ -159,6 +160,132 @@ def github_name(self):
         except SocialProfile.DoesNotExist:
             return None
 
+    """
+    @property
+    def user_cards_in_completed_column(self):
+        from curriculum_tracking.models import AgileCard
+
+        cards_in_completed_column_amount = AgileCard.objects.filter(
+            status=AgileCard.COMPLETE,
+            assignees=self.id
+        ).count()
+
+        return cards_in_completed_column_amount
+    """
+
+    @property
+    def user_cards_in_review_column(self):
```

Comment:
I still think this stuff should be implemented in the curriculum_tracking app instead of inside `core`.

Why do you want it here? Why do we want things to be so tangled up?

Reply:
https://github.com/Umuzi-org/Tilde/pull/183#issuecomment-925282516

Hi Sheena, this relates to my question (https://github.com/Umuzi-org/Tilde/pull/183#issuecomment-925282516).  You will see I moved only the 1st function and all of it's functionality inside core/serializers.py, should it be in curriculum_tracking/serializers.py?